example/Makefile.inc

```diff
@@ -1,6 +1,6 @@
 include $(top_srcdir)/platform/@with_platform@/Makefile.inc
 LIB   = $(top_builddir)/lib
-LDADD = $(LIB)/libodp-linux.la $(LIB)/libodphelper.la
+LDADD = $(LIB)/lib$(ODP_LIB_STR).la $(LIB)/libodphelper.la
```

Comment:
Well... linux-dpdk platform should implement libodp-linux.la, not libodp-dpdk.la. It is an ODP implementation for Linux.